egs/wsj/s5/steps/segmentation/do_asr_sad_data_dir.sh

```diff
@@ -0,0 +1,252 @@
+#! /bin/bash
+
+# Copyright 2016  Vimal Manohar
```

Comment:
This is needed to train on 8kHz data and test on 16kHz, even though we still use only 330Hz to 3000Hz in both cases.